### ISSUE TITLE
update svg.js to link (or mention) this resource

### ISSUE THREAD

**Issue by dasDaniel** [svgdotjs/svgdotjs.github.io#1]
Any chance you could re-direct or update the svgjs.com site 
It wasn't immediately clear, and took a bit of looking around to find out that https://svgdotjs.github.io may be a better resource

**Comment by wout** [comment]
Unfortunately, we no longer have access to svgjs.com.

**Comment by Fuzzyma** [comment]
Any chance we can at least remove the current content from the site? We still have the copyright on the stuff written there...

**Comment by wout** [comment]
Not really. I can ask him again to point the domain at Github. Initially, he just wanted the Domain Tips link under **Recommendations** at the bottom.  But by the time I finished the new site, he had changed his mind.

**Comment by Fuzzyma** [comment]
Thats not what I meant. I think we could forbid him to show anything about svg.js at all (because we have the copyright on this content). So people would at least start looking around for other sources of truth when they realize that svgjs.com has no content about svg.js

**Comment by wout** [comment]
True. I'll contact him again.

**Comment by wout** [comment]
Unfortunately, communication seized. If we want the domain name linked to the new docs, we should at least put up one large banner. Or, we can bug a new name, for which I will gladly pay. Any suggestions?

**Comment by Fuzzyma** [comment]
What do you mean with new name?
At the other hand having one large banner when there is only one link would not hurt so much.
It is not our officical page anymore and we can state that on the site. I guess google will pick this up soon and wont rate it so high anymore.

**Comment by wout** [comment]
With a new domain name, I mean svgjs.net for example or svgdotjs.com. A more top level domain.

In case we would agree to the banner proposal, a large banner will be placed on the home page of the new doc. I would like to have some more visual content there one day and a banner would be very disturbing.

I already changed a lot of links on the web and requested changes on major sites. That's how I got in touch with Chris Coyier when he asked me to write for CSS-Tricks. I think eventually svgjs.com will loose a lot of traffic, probably it already has.
